ScanShop.Mobile/ScanShop.Mobile/ViewModels/QRGenerateViewModel.cs

```diff
@@ -9,15 +9,17 @@ namespace ScanShop.Mobile.ViewModels
 {
     public class QRGenerateViewModel : BaseViewModel
     {
-        private string _QRGenValue = "Hello World!";
-        private int _QRSize = Convert.ToInt32(Math.Round(DeviceDisplay.MainDisplayInfo.Width / DeviceDisplay.MainDisplayInfo.Density * QRSizePercentage / 100));
+        private string _QRGenValue;
+        private int _QRSize;
         private const float QRSizePercentage = 80;
 
         public Command GoBackCommand { get; }
 
         public QRGenerateViewModel()
         {
             GoBackCommand = new Command(OnGoBackClicked);
+            _QRGenValue = "Hello World!";
+            _QRSize = Convert.ToInt32(Math.Round(DeviceDisplay.MainDisplayInfo.Width / DeviceDisplay.MainDisplayInfo.Density * QRSizePercentage / 100));
```

Comment:
This line is too long, better to split the results of mathematical operations into local variables

Reply:
I don't think this is needed, it is only 3 arithmetic operations and we will not use these local variables elsewhere.

Reply:
You are right.
Fixed.